sourcegraph/learn
# Add content: How to commit code with the Git CLI

## Type of change

Add new content

### What should this PR do?

Complete [DEVED-128](https://sourcegraph.atlassian.net/browse/DEVED-128)

### What are the acceptance criteria? 

- The markdown formatting renders correctly
- The front-matter is accurate and complete
- The header image displays correctly on the page and on the content card
- All links are working

### How should this PR be tested?

- Open the deploy preview and check the added content
- Direct link on the deploy preview: https://deploy-preview-200--sourcegraph-learn.netlify.app/how-to-commit-code-with-the-git-command-line-interface

## Pull request process

**Reviewers**:

1. Test functionality using the criteria above.
2. Offer tips for efficiency, feedback on best practices, and possible alternative approaches and things that may not have been considered.
3. For shorter, "quick" PRs, use your best judgement on #​2.
4. Use a collaborative approach and provide resources and/or context where appropriate.
5. Provide screenshots/grabs where appropriate to show findings during review.

**Reviewees**:

1. Prefer incremental and appropriately-scoped changes.
2. Leave a comment on things you want explicit feedback on.
3. Respond clearly to comments and questions.
